src/Examples/Component Examples/Examples.WebAssemblyPlayground/Startup.fs

```diff
@@ -33,5 +24,14 @@ module Program =
 
         let builder = WebAssemblyHostBuilder.CreateDefault(args)
         builder.RootComponents.Add<MainView>("#app")
-        builder.Build().RunAsync() |> ignore
+        let host = builder.Build()
+
+        task {
+            do! AppBuilder.Configure<App>()
+                      .UseReactiveUI()
+                      .StartBlazorAppAsync()
```

Comment:
The WASM startup functions in Avalonia have been made async, and i'm not sure what the best way of handling those is (it currently just ignores the created task like the previous code did)